cylc/rose/__init__.py

```diff
@@ -70,11 +70,37 @@
    for ease of porting Cylc 7 workflows.
 
 
+The ``global.cylc`` file
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cylc Rose Plugin forces the reloading of the ``global.cylc`` file
+to allow environment variables set by Rose to change the global configuration.
+
+For example you could use ``CYLC_SYMLINKS`` as a variable to control
+the behaviour of ``cylc install``:
+
+   .. code-block:: cylc
+
+      #!jinja2
+      # part of a global.cylc file
+      [install]
+         [[symlink dirs]]
+            [[[hpc]]]
+      {% if environ["CYLC_SYMLINKS"] | default("x") == "A" %}
+               run = $LOCATION_A
+      {% elif environ["CYLC_SYMLINKS"] | default("x") == "B" %}
+               run = $LOCATION_B
+      {% else %}
+               run = $LOCATION_C
+      {% endif %}
+
+
+
 Special Variables
 -----------------
 
 The Cylc Rose plugin provides two environment/template variables
```

Comment:
(unrelated to your changes, but just spotted)
```suggestion
The Cylc Rose plugin provides three environment/template variables
```